src/internal/puck/components/ColorPicker.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { SketchPicker, RGBColor, ColorResult } from "react-color";
+import { PresetColor } from "react-color/lib/components/sketch/Sketch";
+
+type ColorPickerProps = {
+  disableAlpha: boolean;
+  color: RGBColor;
+  setColor: (color: RGBColor) => void;
+  presetColors?: PresetColor[] | undefined;
+};
+
+const ColorPicker = (props: ColorPickerProps) => {
+  const { disableAlpha, presetColors, color, setColor } = props;
+  return (
+    <SketchPicker
```

Comment:
I feel like we can just inline this in ColorSelector instead of creating a wrapper that doesn't really do anything.

Reply:
Done